docs/operator/troubleshooting.md

```diff
@@ -0,0 +1,88 @@
+
+In addition to having a look at this section, you want to check [previous issues](https://github.com/aquasecurity/starboard/issues) to see if someone from the community had similar problems before.
+Feel free to either [open an issue](https://github.com/aquasecurity/starboard/issues), reach out on [Slack](https://slack.aquasec.com), or post your questions in the [discussion forum.]()
+
+## Installation
+
+Make sure that the latest version of the Starboard Operator is installed inside of your Kubernetes cluster.
+For this, have a look at the installation [options.](./installation)
+
+For instance, if your are using the Helm deployment, you need to check the Helm Chart version deployed to your cluster:
+```
+helm list -n starboard-system
+```
+
+## Starboard Pod not running
+
+The Starboard Operator will run a pod inside the starboard-system namespace. Make sure that the pod is running correctly:
+
+```
+kubectl get pods -n starboard-system
+```
+
+If the pod is in `failed`, `pending`, or `unknown` check the events and the logs of the pod.
+
+```
+kubectl logs <POD-NAME> -n starboard-system
+```
+
+```
+kubectl describe pod <POD-NAME> -n starboard-system
+```
+
+Note that you have to replace the pod-name with the name of the starboard pod running inside of your cluster.
+
+If your pod is not running, try to look for errors as they can give an indication on the problem.
+
+If there are too many logs messages, try deleting the Starboard pod and observe its behaviour upon restarting. A new pod should spin up automatically after deleting the failed pod.
+
+## ImagePullBackOff or ErrImagePull
+
+Check the status of the Starboard Operator pod running inside of your Kubernetes cluster. If the Status is ImagePullBackOff or ErrImagePull, it means that the Operator either
+
+* tries to access the wrong image
+* cannot pull the image from the registry
+
+Make sure that you are providing the right resources upon installing the Starboard Operator.
+
+## CrashLoopBackOff
+
+If your pod is in `CrashLoopBackOff`, it is likely the case that the pod cannot be scheduled on the Kubernetes node that it is trying to schedule on.
+In this case, you want to investigate further whether there is an issue with the node. It could for instance be that the node does not have sufficient resources.
+
+
+## Reconsilation error
+
+It could happen that the pod appears to be running normally but does not reconcile the resources inside of your Kubernetes cluster.
+
+Check the logs for reconcilation errors:
+```
+kubectl logs <POD-NAME> -n starboard-system
+```
+
+If this is the case, the Starboard Operator likley does not have the right configurations to access your resource. 
+
+## Operator does not create Vulnerability Reports
+
+Every VulnerabilityReport is linked to a [deployment.](./index.md) In case the Starboard Operator does not create a VulnerabilityReport for your workloads, it could be that it is not monitoring the namespace that your workloads are running on.
```

Comment:
To be precise VulnerabilityReports are owned by (or controlled by) immediate Kubernetes workload. So for deployments this would be active ReplicaSet aka current revision.